src/lib/transaction_witness/transaction_witness.mli

```diff
@@ -109,4 +109,5 @@ type t =
 
 val read_all_proofs_from_disk : t -> Stable.Latest.t
 
-val write_all_proofs_to_disk : Stable.Latest.t -> t
+val write_all_proofs_to_disk :
+  proof_cache_db:Proof_cache_tag.cache_db -> Stable.Latest.t -> t
```

Comment:
Nit: It seems that we have the convention to always use labelled argument. Interesting.